sharding-ui/sharding-ui-frontend/package.json

```diff
@@ -9,7 +9,8 @@
     "start": "npm run dev",
     "lint": "eslint --ext .js,.vue src",
     "test": "cross-env NODE_ENV=mock  webpack-dev-server --inline --progress --config build/webpack.dev.conf.js",
-    "build": "node build/build.js"
+    "build": "node build/build.js",
+    "unit": "cross-env BABEL_ENV=test karma start --single-run"
```

Comment:
@wqzwh ，now we use karma，can we remove the easy-mock?

Reply:
Es-mock is suitable for use in the early stages of development, but can not be used for the time being, so this iteration has been deleted.

Karma does unit testing for components and pages